platform/eps/housekeeping.c

```diff
@@ -133,6 +133,13 @@ SAT_returnState hk_report_parameters(HK_struct_id sid, tc_tm_pkt *pkt) {
         cnv32_8( HAL_sys_GetTick(), &pkt->data[1]);
         size += 4;
 
+        //Please add the correct variable
+        pkt->data[size] = rsrc;
```

Comment:
@squarewavesbelow  Please let me know the correct variable that you store the reset source.